packages/utils/package.json

```diff
@@ -28,6 +28,7 @@
     "viem": "catalog:"
   },
   "devDependencies": {
+    "@cartridge/presets": "catalog:",
```

Comment:
is this needed?

Reply:
This is used to pass ERC20 icons to ERC20 class. In the future, it should be provided via gateway API.